model/events/LtiAgsListener.php

```diff
@@ -44,6 +44,8 @@
 
 class LtiAgsListener extends ConfigurableService
 {
+    public const OPTION_AGS_MAX_RETRY = 5;
```

Comment:
I'm not sure it's going to be convenient to configure the service with an option named `5` ;)
Could you please change the value into something more option-like?